test/Java8andUp/playlist.xml

```diff
@@ -921,6 +919,84 @@
 			<subset>SE80</subset>
 		</subsets>
 	</test>
+<!--
+	Java 8 specific test case.
+	Java 9 and above require VM options not available in Java 8.
+-->
+	<test>
+		<featureIds>
+			<featureId>Java access control</featureId>
+		</featureIds>
```

Comment:
Please remove <featureIds>.
We are in the process of removing <featureIds> in all playlist.

Reply:
Done.